packages/eslint-config-base/README.md

```diff
@@ -62,8 +62,8 @@ module.exports = {
 }
 ```
 
-It is also recommended that you lint the whole project folder (ie. `npx eslint .`) instead of just
-some folders (ie. `npx eslint src test`) and create an _.eslintignore_ file excluding any unwanted
+It is also recommended that you lint the whole project folder (that is. `npx eslint .`) instead of just
```

Comment:
Now there is still the extra dot after `is`.

Reply:
Fixed.